### DISCUSSION TITLE
OFX Importer for FireFlyIII

### DISCUSSION THREAD

**Discussion by pelaxa** [firefly-iii/firefly-iii#8550] [General]
I've been using FF3 for a while and while the default importer works well, it requires some setup and maintenance which would not be required for OFX files.  I also do not like anything to be connected to my bank directly so I've been using the CSV import option.

So, I've created and used an OFX importer for a while which makes my life easier.  I've run into various edge cases where the import did not work well and made some improvements but all of this based on my personal use case only.

All this to say that if anyone else wants to give it a shot, it is on GitHub now: https://github.com/pelaxa/ff3-ofx

**Comment by JC5** [comment]
Very cool! Can I mention this in the docs and plug it on Mastodon?

**Reply by pelaxa** [reply]
Thank you for the great application.
Feel free to do as you like.  it is out there and maybe others can help perfect it.
